_includes/nav.njk

```diff
@@ -19,21 +19,9 @@
         <nav class="nav">
             <ul
                 class="menu">
-                {# <li>
-                                <a href="">Learn</a>
-                            </li>
-                            <li>
-                                <a href="">Volunteer</a>
-                            </li>
-                            <li>
-                                <a href="">Support</a>
-                            </li>
-                            <li>
-                                <a href="">About</a>
-                            </li>
-                            <li>
-                                <a href="">Blog</a>
-                            </li> #}
+                <li>
+                    <a href="/about.html">About</a>
+                </li>
```

Comment:
We should use the templating engine that eleventy provides, but I think it's fine to hard-code for now.